Docs/ProjectSpec.md

```diff
@@ -50,6 +50,7 @@ You can also use environment variables in your configuration file, by using `${S
 - [ ] **targetTemplates**: **[String: [Target Template](#target-template)]** - a list of targets that can be used as templates for actual targets which reference them via a `template` property. They can be used to extract common target settings. Works great in combination with `include`.
 - [ ] **packages**: **[String: [Swift Package](#swift-package)]** - a map of Swift packages by name
 - [ ] **localPackages**: **[String]** - A list of paths to local Swift Packages. The paths must be directories with a `Package.swift` file in them. This is used to override `packages` with a local version for development purposes.
+- [ ] **externalProjects**: **[String: [External Project](#external-project)]** - a map of external projects by name
```

Comment:
What about naming this just `projects` of type `ProjectReference`. Thoughts? Or `projectReferences`

Reply:
`projectReferences` seems good! Thanks!